controllers/toolchainconfig/configuration_test.go

```diff
@@ -239,20 +239,12 @@ func TestAutomaticApprovalConfig(t *testing.T) {
 		toolchainCfg := newToolchainConfig(cfg, map[string]map[string]string{})
 
 		assert.False(t, toolchainCfg.AutomaticApproval().IsEnabled())
-		assert.Equal(t, 1000, toolchainCfg.AutomaticApproval().MaxNumberOfUsersOverall())
-		assert.Empty(t, toolchainCfg.AutomaticApproval().MaxNumberOfUsersSpecificPerMemberCluster())
-		assert.Equal(t, 80, toolchainCfg.AutomaticApproval().ResourceCapacityThresholdDefault())
-		assert.Empty(t, toolchainCfg.AutomaticApproval().ResourceCapacityThresholdSpecificPerMemberCluster())
 	})
 	t.Run("non-default", func(t *testing.T) {
-		cfg := commonconfig.NewToolchainConfigObjWithReset(t, testconfig.AutomaticApproval().Enabled(true).MaxNumberOfUsers(123, testconfig.PerMemberCluster("member1", 321)).ResourceCapacityThreshold(456, testconfig.PerMemberCluster("member1", 654)))
+		cfg := commonconfig.NewToolchainConfigObjWithReset(t, testconfig.AutomaticApproval().Enabled(false))
 		toolchainCfg := newToolchainConfig(cfg, map[string]map[string]string{})
 
-		assert.True(t, toolchainCfg.AutomaticApproval().IsEnabled())
-		assert.Equal(t, 123, toolchainCfg.AutomaticApproval().MaxNumberOfUsersOverall())
```

Comment:
shouldn't you check for `CapacityThresholds().MaxNumberOfSpaces` instead?

Reply:
there is only `MaxNumberOfSpacesPerMemberCluster` and not `overall` , this was the original design according to https://issues.redhat.com/browse/ASC-113

Reply:
ok, thanks for the clarification!

Reply:
yes, we decided to drop the "overall" limit because it didn't make any sense. The numbers are set per member cluster every time.